os/board/rtl8730e/src/rtl8730e_boot.c

```diff
@@ -383,6 +383,23 @@ void app_ftl_init(void)
  ****************************************************************************/
 
 #ifdef CONFIG_BOARD_INITIALIZE
+
+static gtimer_t g_timer_core;
+
+void core_status_timer_hdl(void)
+{
+	if (BKUP_Read(BKUP_REG2) == 0x1){
+		ASSERT(0);
+	}
+}
+
+void init_core_status_timer(void)
+{
+	gtimer_init(&g_timer_core, TIMER2);
```

Comment:
Then, can't we use the TIMER2 for user purpose? This is dedicated to Realtek?

Reply:
We have TIMER0~TIMER7, currently KM0 holds TIMER0, and we allocate TIMER2 for this as an AMP crash detector.
Currently, app side only uses TIMER1 for PM. 
In return, we still have TIMER3~TIMER7 for your usage.

Reply:
We have TIMER0~TIMER7, currently KM0 holds TIMER0, and we allocate TIMER2 for this as an AMP crash detector.
Currently, app side only uses TIMER1 for PM. 
In return, we still have TIMER3~TIMER7 for your usage.

Reply:
1. How can user know this?? We should leave something to let user know.
2. Ssusally we use a low number first in an app. How about assigning system dedicated timer to high numbers?

Reply:
Hi @sunghan-chang , your concern make sense. There should be some mechanism to address those issues, but it should be another PR. As the objective of this PR is to handle crash detector for AMP structure.
For the HW timer usage, we will discuss the solution and raise another PR.

Reply:
Hi @sunghan-chang , shall we address this in another PR? Since PM is also using TIMER1, we can discuss on whether to use high numbers for system dedicated timers and standardize the change for both PM and KM4/KM0 crash indication timers in another PR.

Reply:
@jwei5 @edwakuwaku Ok, let's do that with another PR.